devlab/config.ini

```diff
@@ -1,6 +1,6 @@
 # Modify this file with you environment parameters.
 
-scenario_file = devlab/tests/migrate.yaml
+scenario_file = devlab/tests/scenarios/live_migrate.yaml
```

Comment:
It means that by default we will use live_migrate scenario. I think it's wrong. Old one is good, just add a new scenario  file to switch between them.